ofborg/src/maintainers.nix

```
@@ -81,7 +81,10 @@ let
           packageName = pkg.name;
           dueToFiles = pkg.filenames;
         })
-        pkg.maintainers
+        (builtins.filter
+          (maintainer:
+            pkgs.lib.hasAttrByPath ["github"] maintainer)
```

```suggestion
            pkgs.lib.hasAttrByPath [ "github" ] maintainer)
```

Apparently people like these spaces. :sweat_smile:

---

Good point. Solved this in a less subjective manner using `nixfmt`. Ideally, `ofborg` would adopt `nixfmt` for its `.nix` files.